modules/spark-3.2-ext/spark-3.2/src/main/scala/org/apache/ignite/spark/impl/QueryUtils.scala

```diff
@@ -26,6 +26,17 @@ import org.apache.spark.sql.types._
   * Utility class for building SQL queries.
   */
 private[impl] object QueryUtils extends Logging {
+    /**
+      * @param str String.
```

Comment:
This scaladoc does not provide any useful information. Please update.

Reply:
done